manim/cli/render/render_options.py

```diff
@@ -122,16 +130,21 @@ def validate_resolution(ctx, param, value):
         help="Save last frame as png (Deprecated).",
     ),
     option(
-        "-t", "--transparent", is_flag=True, help="Render scenes with alpha channel."
+        "-t",
+        "--transparent",
+        is_flag=True,
+        help="Render scenes with alpha channel.",
```

Comment:
There is a missing `default=None` for this `--transparent` option which can't be set via config file even if added here. This is because the config property for `transparent` uses key `background_opacity` internally to handle the opacity instead of its own. I think `--transparent` should be added to the config file as its own flag which is fit for the title of this PR, but there would need to be a change to address using both `background_opacity` and `transparent` in the config.

Reply:
Initially, I defaulted `transparent` to None but removed it later because it caused problems. Nonetheless, I will look into adding it to the config file.

Reply:
I spent some time trying to add `transparent` flag but unfortunately didn't succeed. I couldn't get tests to pass as all the error messages I received were not that informative. I think we should move forward with this PR without this change. Please note that I removed `png_mode` flag as it is not used anywhere in the library.